.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12.6.0
+      - image: circleci/mongo:4.0.1
+    steps:
+      - checkout
+      - run:
+          name: Code Linting
+          command: |
+            yarn && yarn lint
```

Comment:
Seems you are missing `lint` command in the `scripts` section of `package.json`.

Reply:
Yes, because husky and lint #3 is yet to be set.